pkg/actuators/machine/loadbalancers.go

```diff
@@ -101,3 +79,83 @@ func registerWithNetworkLoadBalancers(client awsclient.Client, names []string, i
 	}
 	return nil
 }
+
+func deregisterNetworkLoadBalancers(client awsclient.Client, names []string, instance *ec2.Instance) error {
```

Comment:
Might be worth a comment on this to explain that we are only deregistering based on IPs because instance ID registrations are automatically removed